### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by adrianbrink**

### CODE REVIEW

File: documentation/docs/src/README.md [hunk 1]
```diff
@@ -4,25 +4,9 @@ Welcome to Namada's docs!
...
 
-[Namada](https://namada.net/) is a sovereign proof-of-stake blockchain, using Tendermint BFT consensus, that enables multi-asset private transfers for any native or non-native asset using a multi-asset shielded pool derived from the Sapling circuit. 
+[Namada](https://namada.net/) is a sovereign proof-of-stake blockchain, using Tendermint BFT consensus, that enables multi-asset private transfers for any native or non-native asset using a multi-asset shielded pool derived from the Sapling circuit.
 
 To learn more about the protocol, we recommend the following resources:
 
 - [Introducing Namada: Shielded Transfers with Any Assets](https://medium.com/anomanetwork/introducing-namada-shielded-transfers-with-any-assets-dce2e579384c)
 - [Namada's specifications](https://specs.namada.net)
-
-### The source
```
Comment (cwgoes) [inline documentation/docs/src/README.md:14]: why remove this? seems useful to me

**Review by cwgoes — APPROVED** [review]
ACK modulo one comment

**Comment by gnosed** [comment]
Is there anything blocking this to be merged @cwgoes @adrianbrink ?

**Comment by cwgoes** [comment]
> Is there anything blocking this to be merged @cwgoes @adrianbrink ?

my comment is still blocking

File: documentation/docs/src/README.md (hunk 1 above)
Comment (tzemanovic) [reply documentation/docs/src/README.md:14]: I added parts back in 7b53c12436609ca10caa72d1a2f7639703276452, but skipped the make commands that are documented in `documentation/README.md`

**Comment by tzemanovic** [comment]
I've hijacked this PR as it's been hanging for a while and finished the renames

**Pull request merged**